packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-color/gux-form-field-color.tsx

```diff
@@ -45,6 +48,14 @@ export class GuxFormFieldColor {
   @Prop()
   labelPosition: GuxFormFieldLabelPosition;
 
+  /**
+   * Field indicator mark which can show *, (optional) or blank
+   * Defaults to required, this shows * for required fields and blank for optional
```

Comment:
Thanks for adding these comments. Super helpful! Is there any way we can we modify the wording on line 53 slightly? 

Maybe:

`Defaults to required, this shows * for required fields and blank for optional` -> `Defaults to required. The component will show * for required fields and blank for optional`